cfme/tests/containers/test_smart_management.py

```diff
@@ -15,14 +15,15 @@
 from cfme.containers.pod import Pod
 from cfme.containers.template import Template
 from cfme.containers.container import Container
+from utils.log import create_sublogger
 from utils.wait import wait_for
 
 pytestmark = [
     pytest.mark.uncollectif(lambda: current_version() < "5.6"),
     pytest.mark.usefixtures('setup_provider'),
     pytest.mark.tier(1)]
 pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
-
+logger = create_sublogger("smart_management")
```

Comment:
Why sublogger and not just default logger (``from utils.log import logger``)?
What is the reason for such verbose logging anyway? Logging should not be a substitute for debugger.

Reply:
The reason for sub-logger is to make the log more clear.
All the test pass perfectly on my private system, but we have many failures in Jenkins  in this area and I want to be able to analyze test and catch the root cause.